### ISSUE TITLE
instr not supported in all versions of sqlite

### ISSUE THREAD

**Issue by bokov** [cattolyst/datafinisher#22]
In lines 251, 264, and 267 of `makesql.py` and in line 62 of `sqldump.cfg` the SQLite command `instr()` is used. This causes...

```
Traceback (most recent call last):
  File "makesql.py", line 404, in <module>
    main(con,csvfile,args.style,dtcp)
  File "makesql.py", line 259, in main
    """)
sqlite3.OperationalError: no such function: instr
```

Production i2b2 has sqlite3 v2.4.1 while on my (AFB) machine the version is 2.6.0 and apparently has `instr` implemented.

Instr might no longer even be necessary due to the `ifgrep` command or some other workaround. In a worst-case scenario it can be implemented as a custom Python command.

**Comment by bokov** [comment]
Apparently the [instr](https://www.sqlite.org/lang_corefunc.html) equivalent in python is the string `find()` method. So, `instr(X,Y)` should give the same result in SQLite as `X.find(Y)` does in Python for cases where `Y` really is found within `X`. When `Y` is not within `X`, the return values is `0` for `instr()` and `-1` for `find()`. For cases where `X` and/or `Y` are `NULL` we'll just need to catch them and return the Python equivalent of `NULL`. There is no need at this time to support non-string `X` or `Y`.

**Comment by dckc** [comment]
I just ran into this. We run python 2.6 (provided by suse enterprise linux)

**Comment by bokov** [comment]
Fixed in latest master. Turns out I didn't even need to use instr anymore. About to test on our own 2.6 installation.

**Comment by bokov** [comment]
Fixed on master in 47146e0e52
